js/script.js

```diff
@@ -159,18 +184,21 @@ function showProject() {
   projectCards.appendChild();
 }
 // eslint-disable-next-line no-unused-vars
-function addBlur() {
-  const blur = document.getElementsByClassName('.container');
-  blur.classList.toggle('blur');
-}
+// function addBlur() {
+//   document.body.classList.add('blur');
+// }
+
+// function removeBlur() {
+//   document.body.classList.remove('blur');
+// }
 
 function togglemobilepopup() {
   const mobilePopup = document.querySelector('.close');
   mobilePopup.classList.toggle('hidden');
-  showProject();
+  // removeBlur();
+  window.location.reload();
 }
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Disabling linters is not really good because it doesn't allow an ethical coding practice that conforms you with industry standards. Kindy remove these lines and see how you can resolve the issues by reading what the message is :wink: